### PR TITLE
chore(contributor): update contributor guidelines with more information

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,145 +1,125 @@
...
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
+    ```
 
-The *type* should always be lowercase as shown below.
+4. You can now run any of this scripts from the root folder.
```
Comment (anescobar1991) [inline CONTRIBUTING.md]: `this` should be `these`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,145 +1,125 @@
...
-```
-and for Pull Requests, use the format:
+Provide the system information which is not limited to the below:
 
-```bash
-Related #37
-```
+- OS: [e.g. macOS, Windows]
+- Browser (if applies) [e.g. chrome, safari]
+- Version of faux-cdn: [e.g. 5.0.0]
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]: is this in all of them?

Comment (nellyk) [reply CONTRIBUTING.md]: yes it is - i've updated this

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,145 +1,125 @@
...
+   > replace `your-github-username` with your github username
 
-```bash
-<type>(<scope>): <subject>
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
    $ npm install
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,145 +1,125 @@
...
 
-*Example*: Transactions may be shown in multiple routes/contexts, but a bug affecting transaction actions may only exist under the "home" route, possibly related to other code. In such cases, you could use the following format:
 
-```bash
-fix(home/transactions): txn actions not working
-```
+## Submitting a new feature
 
-This header makes it clear that the fix is limited in scope to transactions within the home route/context.
+When submitting a new feature request or enhancement of an existing feature please review the following:-
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
When submitting a new feature request or enhancement of an existing feature please review the following:
```

**Pull request merged**